test/server/_fakes/data/services/manage_applications.js

```diff
@@ -0,0 +1,28 @@
+const serviceData = {
```

Comment:
In order to match the name of the service, the file ` test/server/_fakes/data/services/manage_applications.js` should perhaps be called ` test/server/_fakes/data/services/ApplicationDeveloper.js`  ... this should help make clear which service this fakeData belongs to (assuming we had multiple services we were interacting with)